Dockerfile

```diff
@@ -1,38 +1,14 @@
-FROM ubuntu:14.04
-MAINTAINER Colin Alston <colin@praekelt.com>
-RUN apt-get update && apt-get -y --force-yes install libjpeg-dev zlib1g-dev libxslt1-dev libpq-dev nginx redis-server supervisor python-dev python-pip
-RUN apt-get -y install libffi-dev gettext
+FROM praekeltfoundation/django-bootstrap:onbuild
```

Comment:
Rather use the plain image (`praekeltfoundation/django-bootstrap`), not the `:onbuild` variant and copy in the source files later. The `onbuild` thing is cool in that it saves a few steps by automatically copying in your source files but it makes layer caching worse. You want to do a `COPY . /app` as late as possible in the Dockerfile.